CHANGELOG.md

```diff
@@ -36,6 +37,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The double click threshold was raised to `400ms`
 - OSC 52 paste ability is now **disabled by default**; use `terminal.osc52` to adjust it
 - Apply `colors.transparent_background_colors` for selections, hints, and search matches
+- EGL is preferred over GLX when not on Windows or macOs systems
```

Comment:
We can't do that, it'll break the existing systems because transparency doesn't work with EGL and mesa. And probably doesn't work with nvidia either.

Reply:
Fixed :)

Reply:
Mesa 24.1 finally fixed background transparency on EGL on X11. It always worked on Nvidia.